--- AppActivity.java	2016-08-22 16:07:20.000000000 +0800
+++ AppActivity.java	2016-08-22 18:24:31.000000000 +0800
@@ -26,7 +26,47 @@
 package org.cocos2dx.lua;
 
 import org.cocos2dx.lib.Cocos2dxActivity;
+import android.os.Bundle;
 
 
-public class AppActivity extends Cocos2dxActivity{  
+public class AppActivity extends Cocos2dxActivity{
+    
+    private VRSDKWrapper mVRSDKWrapper;
+    
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        mVRSDKWrapper = new VRSDKWrapper();
+        mVRSDKWrapper.onCreate(this);
+    }
+    
+    @Override
+    protected void onDestroy(){
+        mVRSDKWrapper.onDestroy();
+        super.onDestroy();
+    }
+    
+    @Override
+    protected void onResume(){
+        super.onResume();
+        mVRSDKWrapper.onResume();
+    }
+    
+    @Override
+    protected void onPause(){
+        super.onPause();
+        mVRSDKWrapper.onPause();
+    }
+    
+    @Override
+    protected void onRestart(){
+        super.onRestart();
+        mVRSDKWrapper.onRestart();
+    }
+    
+    @Override
+    protected void onStop(){
+        super.onStop();
+        mVRSDKWrapper.onStop();
+    }
 }
