--- AppDelegate.cpp	2016-10-20 11:28:16.000000000 +0800
+++ AppDelegate.cpp	2016-10-20 13:18:35.000000000 +0800
@@ -43,6 +43,8 @@
 #include "cocos/scripting/js-bindings/manual/platform/ios/JavaScriptObjCBridge.h"
 #endif
 
+//VR_PLATFORM_INCLUDES_BEGIN
+//VR_PLATFORM_INCLUDES_END
 USING_NS_CC;
 using namespace CocosDenshion;
 
@@ -76,6 +78,11 @@
         director->setOpenGLView(glview);
 }
 
+//VR_PLATFORM_SOURCES_BEGIN
+    auto vrImpl = new VRGenericRenderer;
+    glview->setVR(vrImpl);
+//VR_PLATFORM_SOURCES_END
+    
     // set FPS. the default value is 1.0/60 if you don't call this
     director->setAnimationInterval(1.0f / 60);
 
