--- AppDelegate.cpp	2016-09-26 14:57:13.000000000 +0800
+++ AppDelegate.cpp	2016-10-20 12:13:04.000000000 +0800
@@ -3,6 +3,8 @@
 #include "audio/include/SimpleAudioEngine.h"
 #include "cocos2d.h"
 #include "scripting/lua-bindings/manual/lua_module_register.h"
+//VR_PLATFORM_INCLUDES_BEGIN
+//VR_PLATFORM_INCLUDES_END
 
 using namespace CocosDenshion;
 
@@ -61,6 +63,12 @@
     //LuaStack* stack = engine->getLuaStack();
     //register_custom_function(stack->getLuaState());
 
+    auto glview = Director::getInstance()->getOpenGLView();
+//VR_PLATFORM_SOURCES_BEGIN
+    auto vrImpl = new VRGenericRenderer;
+    glview->setVR(vrImpl);
+//VR_PLATFORM_SOURCES_END
+    
     if (engine->executeScriptFile("src/main.lua"))
     {
         return false;
