--- AppDelegate.cpp	2016-08-05 11:58:57.000000000 +0800
+++ AppDelegate.cpp	2016-08-05 14:52:10.000000000 +0800
@@ -3,6 +3,8 @@
 #include "audio/include/SimpleAudioEngine.h"
 #include "cocos2d.h"
 #include "scripting/lua-bindings/manual/lua_module_register.h"
+//VR_PLATFORM_INCLUDES_BEGIN
+//VR_PLATFORM_INCLUDES_END
 
 using namespace CocosDenshion;
 
@@ -65,7 +67,12 @@
     {
         return false;
     }
-
+    
+    auto glview = Director::getInstance()->getOpenGLView();
+//VR_PLATFORM_SOURCES_BEGIN
+    auto vrImpl = new VRGenericRenderer;
+    glview->setVR(vrImpl);
+//VR_PLATFORM_SOURCES_END
     return true;
 }
 
